src/lifx/packets/stateTileState64.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+const Packet = {
+  size:
+    1 + 1 + 1 + 1 + 1 +
+    64 * (2 + 2 + 2 + 2),
```

Comment:
The `size` is quite confusing like this. Just use the end result value like on the other packets.

Reply:
I simply changed it to the value.
To put such things in the comments is not helpful.
So we lose the source of this arbitrated value, i can live with this.